src/tests/ftest/util/hdf5_vol_test_base.py

```diff
@@ -61,14 +74,11 @@ def run_test(self):
         # create dfuse container
         self.start_dfuse(self.hostlist_clients, self.pool, self.container)
 
-        # for test_param in test_list:
-        # testname = test_param[0][1]
-        # client_processes = test_param[1][1]
-        exe = os.path.join(test_repo, testname)
+        # Setup the job manager
         if mpi_type == "openmpi":
-            manager = Orterun(exe, subprocess=False)
+            manager = Orterun(self.hdf5_vol_cmd)
         else:
-            manager = Mpirun(exe, subprocess=False, mpitype="mpich")
+            manager = Mpirun(self.hdf5_vol_cmd, mpitype="mpich")
```

Comment:
(pylint-redefined-variable-type) Redefinition of manager type from job_manager_utils.Orterun to job_manager_utils.Mpirun